### PR TITLE
Configure and activate SauceLabs for AMP.

### PR DESCRIPTION
**Pull request by cramforce**
Currently only runs our existing tests in Chrome. But this is now super easy to extend to other browsers.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -61,7 +61,11 @@ module.exports = function(config) {
       Chrome_travis_ci: {
         base: 'Chrome',
         flags: ['--no-sandbox']
-      }
+      },
+      SL_Chrome: {
+        base: 'SauceLabs',
+        browserName: 'chrome'
+      },
```
Comment (jmadler) [inline karma.conf.js]: extra comma -- not sure if JS cares

Comment (erwinmombay) [inline karma.conf.js]: intentional. i'll probably enforce it through the linter soon =o

Comment (cramforce) [inline karma.conf.js]: There is a missing comma after 'chome' :)

Comment (erwinmombay) [inline karma.conf.js]: hah, that too

**Comment by cramforce** [comment]
I will merge this. It should currently do very little.

**Pull request merged**